source/behavior/entities/format/events.json

```diff
@@ -126,6 +126,25 @@
             }
           }
         },
+        "emit_vibration": {
+          "title": "Emit Vibration",
+          "description": "",

```

Comment:
Best to mark it, haven't found any official text on it yet

```suggestion
          "description": "UNDOCUMENTED",
```

Reply:
Only place I've seen so far it is the 1.20.60 changelogs